tests/bluetooth/tester/src/audio/btp_vcp.c

```diff
@@ -503,22 +540,9 @@ static void set_register_params(uint8_t gain_mode)
 
 uint8_t tester_init_vcs(void)
 {
-	int err;
-
 	set_register_params(BT_AICS_MODE_MANUAL);
 
-	err = bt_vcp_vol_rend_register(&vcp_register_param);
-	if (err) {
-		return BTP_STATUS_FAILED;
-	}
-
-	err = bt_vcp_vol_rend_included_get(&included);
-	if (err) {
-		return BTP_STATUS_FAILED;
-	}
-
-	aics_server_instance.aics_cnt = included.aics_cnt;
-	aics_server_instance.aics = included.aics;
+	vcs_registered_flag = false;
```

Comment:
Given that `vcs_registered_flag` is `static`, this should already be false at this point

```suggestion
```

Reply:
Done